script/umi_ftr.txt

```diff
@@ -28849,6 +28849,7 @@ return
 	if $Free3 == "cn" mov $Free3,"Chinese Simp."
 	if $Free3 == "cht" mov $Free3,"Chinese Trad."
 	if $Free3 == "it" mov $Free3,"Italiano"
+	if $Free3 == "tr" mov $Free3,"Witch Love"
```

Comment:
I would suggest to use the actual language name here, it is not very intuitive otherwise.